setup.py

```diff
@@ -28,7 +29,7 @@ def read_long_description_from_readme():
 
 setup(
     name="openstf",
-    version="2.2.1",
+    version="2.2.2",
```

Comment:
Good idea to make a new release, but I'm wondering if including the increase in version number in the PR is the workflow we desire for this. 
However, this is very practical:)

Reply:
What would you suggest? Or what would be an alternative workflow? Merging to master first and then incrementing the version number? The disadvantage of that strategy would be to have commits which are not releases on master. I would prefer that every commit is a new version on master. Or did you have an other alternative strategy in mind, happy to learn:)

Reply:
Alternative would be the workflow you describe. That's more or less what typically happens now.

Reply:
I believe (not totally sure) that incrementing the version before merging is more common in both in Gitflow or Githubflow (most images indicate a version number for every commit/merge on master). After the merge a new release is ready in that case. I would suggest we do just that. But its not a very big deal if someone forgets and bumps the version on main. (You could also always just do it on the already merged feature branch and merge it again ofc).